release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3366,6 +3366,8 @@ Contact Red Hat Support for additional details and assistance.
 The conversion to the new OVN Kubernetes interconnect multi-zone architecture can lead to packet drops that cause brief network outages. 
 The impact is on East/West traffic in local gateway mode with `routingViaHost` set to `true`. (link:https://issues.redhat.com/browse/OCPBUGS-38891[*OCPBUGS-38891*])
 
+* There is a known issue with HAProxy when updating a cluster to {product-title} {product-version} that duplicates encoding headers. This results in losing external access to pods when they expose a route that uses these duplicated headers. For more details and a workaround, see the information in this link:https://access.redhat.com/solutions/7055002[Red Hat Knowledgebase article]. (link:https://issues.redhat.com/browse/OCPBUGS-43095[*OCPBUGS-43095*])
```

Comment:
The wording here is a bit confusing in a few places.  
- Is "workaround" the correct term?  The solution is to fix the application to comply with HTTP specifications.  We're not providing any kind of accommodation in OpenShift for invalid headers; we're providing a warning and instructions on what needs to be fixed in the application.  
- The issue here is specifically with response headers sent by the application (not _request_ headers, and not headers that anything should _use_, as they are invalid).  
- I think it would be clearer to write "multiple" or "invalid" `transfer-encoding` headers since sending `transfer-encoding: chunked` and `transfer-encoding: xyz` is just as problematic as sending `transfer-encoding: chunked` twice.  
```suggestion
* There is a known issue with HAProxy in {product-title} {product-version} involving applications that send invalid `Transfer-Encoding` headers. This issue results in losing external access to pods when they expose a route that sends these invalid headers. For more details, see the information in this link:https://access.redhat.com/solutions/7055002[Red Hat Knowledgebase article]. (link:https://issues.redhat.com/browse/OCPBUGS-43095[*OCPBUGS-43095*])
```

Reply:
thanks Miciah!